common/scripts/create-component.sh

```diff
@@ -114,6 +118,15 @@ for devenv in dev test ; do
         "--param=ENV=${devenv}"
         )
 
+    if [ ! -z "${QSBASE}" ]; then
+      for envfile in "${QSBASE}/ocp-$devenv.env" "${QSBASE}/ocp.env" ; do
```

Comment:
What could the usecase for different config for dev/test be? Not sure I would add this distinction without a usecase ...

Reply:
I will make another adjustment for that.

Reply:
This is now addressed as well.